bluechip/chain.json

```diff
@@ -0,0 +1,74 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "bluechip",
+    "status": "live",
+    "website": "https://www.bluechip.link",
+    "network_type": "mainnet",
+    "pretty_name": "BlueChip",
+    "chain_id": "bluechip_1",
+    "bech32_prefix": "bcp",
```

Comment:
Pending registration: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
I have made a pull request on the registry

Reply:
I see it registered as "bluechip", not as "bcp". Please make sure they are both the same and match what the chain uses.

Reply:
I misread the column. I have updated my information on SLIP table.